docs/REST API/rest-api.md

```diff
@@ -4541,26 +4539,26 @@ Content-Type: application/octet-stream
 
 `Status: 200`
 
-### Get metadata about a bootstrap package
 
-_Available in Fleet Premium_
+### Get or download a bootstrap package
 
-Get information about a bootstrap package that was uploaded to Fleet.
+_Available in Fleet Premium_
 
-`GET /api/v1/fleet/mdm/apple/bootstrap/{team_id}/metadata`
+`GET /mdm/bootstrap/{team_id}`
 
 #### Parameters
 
-| Name       | Type    | In    | Description                                                                                                                                                                                                        |
-| -------    | ------  | ---   | ---------------------------------------------------------------------------------------------------------------------------------------------------------                                                          |
-| team_id    | string  | url   | **Required** The team ID for the package. Zero (0) can be specified to get information about the bootstrap package for hosts that don't belong to a team.                                                          |
-| for_update | boolean | query | If set to `true`, the authorization will be for a `write` action instead of a `read`. Useful for the write-only `gitops` role when requesting the bootstrap metadata to check if the package needs to be replaced. |
+| Name       | Type    | In    | Description                                      |
+| -----------| ------  | ----- | ------------------------------------------------ |
+| team_id    | integer | path  | **Required.** The desired bootstrap package's team's ID. |
+| alt        | string  | query | If specified and set to "media", downloads the bootstrap package's contents. |
+| for_update | boolean | query | If set to `true`, the authorization will be for a `write` action instead of a `read`. Useful for the write-only `gitops` role when requesting the bootstrap metadata to check if the package needs to be replaced. _(Only relevant if `alt` is not specified.)_ |
```

Comment:
@noahtalerman wasn't sure if this parameter was still needed so kept it for now. Let me know if I should remove it.

Reply:
@roperzh is this parameter still needed?